redact/test/index.test.ts

```diff
@@ -0,0 +1,159 @@
+import { describe, expect, test } from "@jest/globals";
+import { redact } from "../index";
+
+describe("ArcjetRedact", () => {
+  describe("redact()", () => {
+    test("it will do nothing if no entities are configured", async () => {
+      const text = "email test@example.com phone 011234567 ip 10.12.234.2";
+      const [redacted] = await redact(text, { redact: [] });
```

Comment:
Don't know feel like this logic should be swapped? I feel like you should be able to use this library like `const [redacted] = await redact(text);` and have it apply all our default redactions.

Reply:
Do nothing, but log a warning to say that no redactions were configured?

Reply:
What if we default to redact all if `entities` (renamed from `redact`) is undefined? If they pass in an empty list then we should warn and redact nothing.

Reply:
What is the purpose of using a redact library and redacting nothing? You should just remove the call to the library in that case. I think we'd throw on an empty list.

Reply:
Throw or log a warning, either is fine by me. There is no purpose of using a redact library and redacting nothing so the goal is to guard against user error by letting them know they made a mistake.

Reply:
Makes sense, I'll update it to throw if someone passes an empty list. It currently redacts all if the list isn't provided at all.